hr_timesheet_sheet/models/hr_timesheet_sheet.py

```diff
@@ -567,15 +567,25 @@ class SheetLine(models.TransientModel):
         default=0,
     )
 
+    def _is_positive(self, value, equal=False):
+        if value > 0.0:
+            return True
+        elif value == 0.0 and equal:
+            return True
+        else:
+            return False
+
+    def _negative_unit_amount(self):
+        """Dummy method for the tests"""
+        return True
+
     @api.onchange('unit_amount')
     def onchange_unit_amount(self):
         """This method is called when filling a cell of the matrix.
         It checks if there exists timesheets associated  to that cell.
         If yes, it does several comparisons to see if the unit_amount of
         the timesheets should be updated accordingly."""
         self.ensure_one()
-        if self.unit_amount < 0.0:
```

Comment:
Better extract it to "normalize unit amount" something

Reply:
?

Reply:
`self.unit_amount = self._normalize_unit_amount(self.unit_amount)` ?

Reply:
or `adjust_unit_amount`, as extension point

Reply:
`adjust_unit_amount` is an empty method?